### ISSUE TITLE
[Woo POS] Make POSModals accessible

### ISSUE THREAD

**Issue by joshheald** [woocommerce/woocommerce-ios#13668]
Extracted from #13517 

POS modals should be the only accessible view via VoiceOver when shown – nothing below the curtain should be announced.

**Comment by dangermattic** [comment]
Thanks for reporting! 👍
